.config/babel.config.json

```diff
@@ -1,8 +1,4 @@
 {
-  "plugins": [
-    "@babel/plugin-syntax-top-level-await",
-    "@babel/plugin-syntax-jsx"
```

Comment:
was it remark that was using this?

moment

Reply:
🔗 https://mdxjs.com/packages/remark-mdx/#authoring

was thinking about JSX, but will keep it around anyway